presidio-analyzer/app.py

```diff
@@ -135,8 +135,9 @@ def supported_entities() -> Tuple[str, int]:
         def http_exception(e):
             return jsonify(error=e.description), e.code
 
+server = Server()
```

Comment:
Might be nitty picking, but I'm curious on what is the difference between instantiating Server here compared to within the `if name == main`. For instance, if I import app.py, the Server instance would be instantiated. I'm not sure if importing app.py is a use case, but it would be a side effect of this change in case somebody does this anywhere. WDYT?

Reply:
Thats a great pickup. When running with Gunicorn, it doesn't execute the code in the`__main__` block. It just imports the module and looks for the app object to serve the application, which is why I moved the server code out of the `__main__` block.

However, I didn't remove the `__main__` block entirely, so if anyone wants to run just the flask development server they still have to ability to do so.

To avoid your concern of instantiating the server on import of app.py, we can avoid it by doing something like this:
```python
def create_app():
    server = Server()
    return server.app

if __name__ == "__main__":
    app = create_app()
    port = int(os.environ.get("PORT", DEFAULT_PORT))
    app.run(host="0.0.0.0", port=port)
```

We can then run Gunicorn in the Dockerfile like this:
```
CMD poetry run gunicorn -w $WORKERS -b 0.0.0.0:$PORT 'app:create_app()'
```

This should ensure that if anyone is just importing app.py it won't create the server instance. If you're happy with this, I'll commit the changes

Reply:
Thanks! This is a better solution IMHO so let's go with it.